app/models/king.rb

```diff
@@ -1,4 +1,24 @@
 class King < Piece
-	def move
+
+	def valid_move?(start_point, end_point)
+		# Define starting and ending coordinates for pieces
+		start_x, start_y = start_point
+		end_x, end_y = end_point
+
+		#determine if move is only 1 space in any direction
+		if (start_x - end_x).abs <= 1 && (start_y - end_y).abs <= 1
+			#then check if space is occupied
```

Comment:
so I think if just 

return (start_x - end_x).abs <= 1 && (start_y - end_y).abs <= 1

you should be all set.